<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>760</width>
    <height>579</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>421</width>
      <height>151</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayoutInfo">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QComboBox" name="comboBoxName"/>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_3" rowstretch="0,0,1,0" columnstretch="0,2">
         <property name="rightMargin">
          <number>5</number>
         </property>
         <item row="2" column="0">
          <widget class="QLabel" name="labelComments">
           <property name="text">
            <string>Comments:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="labelId">
           <property name="text">
            <string>Hardware ID:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="labelIdVal">
           <property name="text">
            <string>0000-0000-0000-0000</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="labelUploadedByVal">
           <property name="text">
            <string>Jos√© Puga</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="labelUploadedBy">
           <property name="text">
            <string>Uploaded by:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="labelCommentsVal">
           <property name="text">
            <string>Lorem ipsum dolor sit amen. Consectetuer edipicin est. Esto es una mierda en un palo.</string>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget_3">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>20</y>
      <width>331</width>
      <height>501</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayoutImage">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <spacer name="horizontalSpacer_3">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QLabel" name="labelImage">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>256</width>
           <height>340</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>256</width>
           <height>340</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="pixmap">
          <pixmap>images/test3.png</pixmap>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer_3">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButtonSetDefaults">
         <property name="text">
          <string>Reset to &amp;Defaults</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonCancel">
         <property name="text">
          <string>&amp;Cancel</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButtonApply">
         <property name="text">
          <string>&amp;Apply</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>421</width>
      <height>331</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayoutButtons" stretch="3,2">
     <item>
      <widget class="QGroupBox" name="groupBoxCurrentLayout">
       <property name="title">
        <string>Layout to set:</string>
       </property>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>175</width>
          <height>362</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,3">
         <property name="sizeConstraint">
          <enum>QLayout::SetMaximumSize</enum>
         </property>
         <property name="leftMargin">
          <number>15</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <item row="8" column="0">
          <widget class="QRadioButton" name="radioButtonBtn9">
           <property name="text">
            <string>&amp;9</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QComboBox" name="comboBoxBtn9"/>
         </item>
         <item row="6" column="1">
          <widget class="QComboBox" name="comboBoxBtn7"/>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="comboBoxBtn4"/>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="comboBoxBtn3"/>
         </item>
         <item row="7" column="1">
          <widget class="QComboBox" name="comboBoxBtn8"/>
         </item>
         <item row="7" column="0">
          <widget class="QRadioButton" name="radioButtonBtn8">
           <property name="text">
            <string>&amp;8</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QRadioButton" name="radioButtonBtn4">
           <property name="text">
            <string>&amp;4</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QComboBox" name="comboBoxBtn6"/>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="comboBoxBtn1"/>
         </item>
         <item row="0" column="0">
          <widget class="QRadioButton" name="radioButtonBtn1">
           <property name="text">
            <string>&amp;1</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="comboBoxBtn5"/>
         </item>
         <item row="4" column="0">
          <widget class="QRadioButton" name="radioButtonBtn5">
           <property name="text">
            <string>&amp;5</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QRadioButton" name="radioButtonBtn3">
           <property name="text">
            <string>&amp;3</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="comboBoxBtn2"/>
         </item>
         <item row="1" column="0">
          <widget class="QRadioButton" name="radioButtonBtn2">
           <property name="text">
            <string>&amp;2</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QRadioButton" name="radioButtonBtn7">
           <property name="text">
            <string>&amp;7</string>
           </property>
           <property name="checkable">
            <bool>true</bool>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QRadioButton" name="radioButtonBtn6">
           <property name="text">
            <string>&amp;6</string>
           </property>
           <attribute name="buttonGroup">
            <string notr="true">buttonGroup</string>
           </attribute>
          </widget>
         </item>
         <item row="9" column="1">
          <spacer name="verticalSpacer_4">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBoxDefaultLayout">
       <property name="title">
        <string>Default Layout:</string>
       </property>
       <widget class="QWidget" name="gridLayoutWidget_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>30</y>
          <width>160</width>
          <height>367</height>
         </rect>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="topMargin">
          <number>15</number>
         </property>
         <property name="verticalSpacing">
          <number>14</number>
         </property>
         <item row="2" column="1">
          <widget class="QLabel" name="labelDefault3">
           <property name="text">
            <string>3 - BTN_NONE</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>6</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLabel" name="labelDefault7">
           <property name="text">
            <string>7 - BTN_NONE</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>9</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>7</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QLabel" name="labelDefault9">
           <property name="text">
            <string>9 - BTN_NONE</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="labelDefault2">
           <property name="text">
            <string>2 - BTN_NONE</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>2</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLabel" name="labelDefault6">
           <property name="text">
            <string>6 - BTN_NONE</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>4</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="labelDefault5">
           <property name="text">
            <string>5 - BTN_NONE</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_1">
           <property name="text">
            <string>1</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>3</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>5</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="labelDefault1">
           <property name="text">
            <string>1 - BTN_NONE</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="labelDefault4">
           <property name="text">
            <string>4 - BTN_NONE</string>
           </property>
          </widget>
         </item>
         <item row="7" column="1">
          <widget class="QLabel" name="labelDefault8">
           <property name="text">
            <string>8 - BTN_NONE</string>
           </property>
          </widget>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>8</string>
           </property>
          </widget>
         </item>
         <item row="9" column="1">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="9" column="0">
          <spacer name="verticalSpacer_5">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>760</width>
     <height>29</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="enabled">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>comboBoxName</tabstop>
  <tabstop>radioButtonBtn1</tabstop>
  <tabstop>comboBoxBtn1</tabstop>
  <tabstop>radioButtonBtn2</tabstop>
  <tabstop>comboBoxBtn2</tabstop>
  <tabstop>radioButtonBtn3</tabstop>
  <tabstop>comboBoxBtn3</tabstop>
  <tabstop>radioButtonBtn4</tabstop>
  <tabstop>comboBoxBtn4</tabstop>
  <tabstop>radioButtonBtn5</tabstop>
  <tabstop>comboBoxBtn5</tabstop>
  <tabstop>radioButtonBtn6</tabstop>
  <tabstop>comboBoxBtn6</tabstop>
  <tabstop>radioButtonBtn7</tabstop>
  <tabstop>comboBoxBtn7</tabstop>
  <tabstop>radioButtonBtn8</tabstop>
  <tabstop>comboBoxBtn8</tabstop>
  <tabstop>radioButtonBtn9</tabstop>
  <tabstop>comboBoxBtn9</tabstop>
  <tabstop>pushButtonSetDefaults</tabstop>
  <tabstop>pushButtonCancel</tabstop>
  <tabstop>pushButtonApply</tabstop>
 </tabstops>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
